src/app/shared/view-issue/view-issue.component.ts

```diff
@@ -37,7 +37,7 @@ export enum ISSUE_COMPONENTS {
 export class ViewIssueComponent implements OnInit, OnDestroy {
   issue: Issue;
   isIssueLoading = true;
-  isCommentsLoading = true;
+  isCommentsLoading = false;
```

Comment:
What is the reason for changing the default value of this? Is this Boolean attribute still in use? If not we can remove it.